README.rst

```diff
@@ -171,6 +171,8 @@ In e.g. your ``init.lua``:
         infoview = {
             -- Use a wider or narrower infoview.
             width = 50,
+            -- automatically open infoview on new tab?
```

Comment:
This isn't on new tab right (I think you changed that since yesterday, that behavior was surprising to me, so I much prefer this way) -- it's now when entering a lean buffer, like it was before this PR?

If so is there really a reason someone would have `enable = truthy` and still `autoopen = false`? Can't those just be the same? Either the infoview is enabled (and then it autoopens) or not?

Reply:
Yep, now `ensure_open()` only triggers on BufEnter. However, initialization only happens on new tab currently (until we get to per-window), which is where this option is considered, so the comment is accurate.

`enable = false` would disable infoview autocmds, though this does seem rather internal so we could make the behavior to always enable, and use the `enable` field to replace `autoopen`.

Reply:
(see recent commit)

Reply:
Oh I see what you're saying - yeah, we should say something along the lines of "automatically open infoview on entering lean buffer in new tab" instead

Reply:
I guess I have to look at this PR a lot closer, I don't follow yet what it does really.

But my question was less about the name -- we have `infoview = false` already for "I don't want an infoview". What I was asking was what this option did beyond that

Reply:
> Oh I see what you're saying - yeah, we should say something along the lines of "automatically open infoview on entering lean buffer in new tab" instead

Right now the comment makes sense to me at least, but I'm skeptical someone will ever want `infoview = not false` and `autoopen = false`, or do you see a scenario for that?

Reply:
This is for someone who's using A LOT of tabs/windows, so they may want to periodically control whether their new tabs/windows automatically open infoviews. Perhaps a bit rare, though I figured since the refactor (intended for #74) enables this anyways we might as well expose it.